apps/engine/apache-arrow-js-bundle/README.md

```diff
@@ -4,10 +4,12 @@
 
 ### Building
 
-Run
+Move to the package folder and run:
 
-- `yarn install`
-- `yarn workspace @hashengine/apache-arrow-js-bundle bundle`
+```sh
+npm install
+npm run bundle
```

Comment:
What's the reasoning behind using `npm` here? That feels very strangely inconsistent with the rest of the repository.

I appreciate we don't want it to be a workspace within the root one, but we can avoid that by just not including it in the paths of all the workspaces surely? I don't see why we have to revert to npm here

Reply:
A saw `npm` and `package-lock.json` in `apps/engine/stdlib`, so decided to do the same in this sibling folder. Both packages are ‘islands’, i.e. the are not visible from the root yarn workspace.

Reply:
I was choosing between 'upgrading' `stdlib` to `yarn` and 'downgrading' `apache-arrow-js-bundle` to npm. The latter option won as I found it less risky. I fully agree with you about consistency, I’m just trying to make changes to `engine` as safe as possible.